### PR TITLE
Standardize more labels

### PR DESCRIPTION
**Pull request by annevk**
This standardizes a bunch of additional labels as proposed in #80 and makes it easy to add them to a new repository by providing a script that takes care of that.

### CODE REVIEW

**Comment by domenic** [comment]
I think GitHub did add a new feature to add descriptions to labels pretty recently. I definitely think we should use it if at all possible.

**Comment by domfarolino** [comment]
Do you think it would be worth specifying *web-platform-* tests for needs-tests, or no?

**Comment by domfarolino** [comment]
GitHub does allow label descriptions. For example, clicking `edit` on any of the labels in https://github.com/whatwg/console/labels allows an optional description.

**Comment by annevk** [comment]
@domfarolino maybe in the description.

@domenic where does it surface? Would have to fiddle with the script somewhat to make it update existing repositories as well. And would have to figure out a way to put 3 fields in this ham-fisted associative array.

**Comment by domfarolino** [comment]
@annevk Indeed, that's what I was referring to. I think WPTs could be mentioned there since that's often what we mean, but either way :)

**Comment by annevk** [comment]
Current status: created `labels.json` to become the authoritative source. `labels.py` takes that and generates `LABELS.md`. `labels.sh` needs to be rewritten in Python still and aside from creating also modify existing labels (basically do two network requests it seems, GitHub doesn't offer set).

I'd appreciate review, color suggestions for labels still lacking a color, and if someone wants to write the remaining Python be my guest.

**Comment by annevk** [comment]
I think this is now in a state where we could land this. The script works.

Note that I had to shorten some descriptions to not hit GitHub's limit (which seems to be about a 100 characters).

**Comment by annevk** [comment]
(I am thinking though that perhaps we should take this further and automate things completely through Travis, depending on how often we plan to add new global labels...)

**Comment by annevk** [comment]
I ran the script for MIME Sniffing and it seems to work great: https://github.com/whatwg/mimesniff/labels. I think I'd prefer landing this and then iterating further.

File: labels.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    "url_exclude_is_open": true
+  },
+  {
+    "name": "clarification",
+    "description": "Standard could be clearer",
+    "color": "fef2c0"
+  },
+  {
+    "name": "compat",
+    "description": "Standard is not web compatible",
```
Comment (domenic) [inline labels.json]: Sometimes we use this for cases where something needs to be standardized so browsers can become web compatible, I think?

File: labels.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  },
+  {
+    "name": "good first issue",
+    "description": "Ideal for someone new to a WHATWG standard or software project",
+    "color": "bfe5bf"
+  },
+  {
+    "name": "i18n-alreq",
+    "description": "Used by the internationalization community to notify Arabic script experts of relevant issues",
+    "color": "d4c5f9"
```
Comment (domenic) [inline labels.json]: I think it'd be nice to use something different than the addition/proposal color

File: labels.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  },
+  {
+    "name": "impacts documentation",
+    "description": "Used by documentation communities, such as MDN, to track changes that impact documentation",
+    "color": "ececec",
+    "url_exclude_is_open": true
+  },
+  {
+    "name": "interop",
+    "description": "Implementations are not compatible",
```
Comment (domenic) [inline labels.json]: s/compatible/interoperable. Maybe add "with each other"?

File: labels.json [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    "color": "1cddc7"
+  },
+  {
+    "name": "needs implementer interest",
+    "description": "Moving the issue forward requires implementers to express interest",
+    "color": "5319e7"
+  },
+  {
+    "name": "needs tests",
+    "description": "Moving the issue forward requires someone to write tests",
```
Comment (domenic) [inline labels.json]: Usually pull requests, not issues, right? (Although it seems like in general you're trying to encompass both with the word "issue", which is probably fine in general.)

File: LABELS.md [hunk 5]
```diff
@@ -2,11 +2,28 @@
 
 There is currently no WHATWG-wide label policy, except for:
```
Comment (domenic) [inline LABELS.md]: I wonder if this intro sentence is no longer very accurate...

**Comment by domfarolino** [comment]
Ah sorry I was late to the party on this one. This is really awesome thank you so much

**Pull request merged**